backend/src/main/java/io/github/orionhealth/xbdd/util/LoggedInUserUtil.java

```diff
@@ -25,8 +25,10 @@ public static User getLoggedInUser() {
 
 			// Github-specific attributes
 			user.setUserId(String.format("%s-%s", user.getLoginType(), principal.getName()));
-			user.setSocialLogin(principal.getAttribute("login"));
-			user.setDisplay(principal.getAttribute("name"));
+			final String socialLogin = principal.getAttribute("login");
+			user.setSocialLogin(socialLogin);
+			final String display = principal.getAttribute("name");
+			user.setDisplay(display != null ? display : socialLogin);
```

Comment:
I just did some tests with github, the github will trim the space, so I won't get the blank display.
One more thing is github will give a random profile avatar if you don't provide one, so technically we shouldn't have any case from github user that will use/break the avatar logic, but it broke when I tested, the reason is the getInitials() and getHSLFromString() is unnecessarily call.
There is actually a bug that could break the avatar logic if the user doesn't have an avatar url (currently it could only happen with our dummy user). We didn't trim the display and it could break the string avatar when we have a username (' ke vin').

It just occurred to me that you have asked the same question about !== last time, and we found it is not js file and you removed the question from the comment.